elasticsearch.md

```diff
@@ -1738,7 +1738,11 @@ The `groups` rule accepts a list of group names. This rule will match if the res
       auth_key: alice:p455phrase
 
     - username: "bob"
-      groups: ["team2", "team4"]
+      groups: 
+      - id: "team2"
+        name: "Team 2"       
```

Comment:
IMO we need to say a word what's this. And add the link to the details section too.